Libraries/WebSocket/WebSocket.js

```diff
@@ -53,9 +53,9 @@ class WebSocket extends WebSocketBase {
     RCTWebSocketModule.send(message, this._socketId);
   }
 
-  sendArrayBufferImpl(): void {
-    // TODO
-    console.warn('Sending ArrayBuffers is not yet supported');
+  sendArrayBufferImpl(data: ArrayBuffer): void {
```

Comment:
We should be trivially able to also handle the `ArrayBufferView` case here, by checking for `ArrayBuffer.isView(data)` and doing `data = data.buffer` in that case. Of course we'd need to amend the check in `send()` that sends us here to `sendArrayBufferImpl()` as well.